web/lib/components/word/Speech.jsx

```diff
@@ -1,26 +1,45 @@
+import React from 'react'
 import { getVoiceSettings } from '../../storage'
 import { VolumnIcon } from '../common/Icons'
 
-const LANG = 'en-GB'
+const LANG = 'en'
 
 export function Speech({ word }) {
-  const doSpeak = () => {
-    const selectedVoice = getVoiceSettings().get()
+  const [voices, setVoices] = React.useState(null)
+
+  React.useEffect(() => {
+    doGetVoices()
 
+    if (window.speechSynthesis.onvoiceschanged !== undefined) {
+      window.speechSynthesis.onvoiceschanged = doGetVoices
+    }
+  }, [])
```

Comment:
Use `window.speechSynthesis.addEventListener` instead. And don't forget to `.removeEventListener` as a clean up function of effect.